src/app/components/Filters/FieldsetList.jsx

```diff
@@ -18,6 +18,36 @@ class FieldsetList extends React.Component {
     this.state = {
       values: updatedFilterValues,
     };
+
+    this.onFilterClick = this.onFilterClick.bind(this);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    const {
```

Comment:
Update your PR with the latest from `filter-feature`. This has since been updated.

Reply:
It is the latest update.

Reply:
The latest update is different: https://github.com/NYPL-discovery/discovery-front-end/blob/filter-feature/src/app/components/Filters/FieldsetList.jsx